systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -85,6 +86,14 @@ public interface Constants {
     String ECHO_SINK_FILE_NAME = "echo-sink-test.jar";
     String ECHO_SINK_JAR_WRONG_CHECKSUM = "f1f167902325062efc8c755647bc1b782b2b067a87a6e507ff7a3f6205803220";
 
+
+    String NODE_BROKER_CONFIG_HASH_ANNOTATION = ResourceLabels.STRIMZI_DOMAIN + "broker-configuration-hash";
```

Comment:
Extra line + maybe you can move the annotation from `KafkaCluster` class to some common, so we can use it as well (as other annotations). WDYT @scholzj ?